jdaviz/app.py

```diff
@@ -102,9 +102,15 @@ def to_unit(self, data, cid, values, original_units, target_units):
         # as the original native units of the component in the data.
         if cid.label == "flux":
             spec = data.get_object(cls=Spectrum1D)
-            eqv = u.spectral_density(spec.spectral_axis)
+            if len(values) == 2:
```

Comment:
This avoids an array shape mismatch in the unit conversion code for the y-limits case, which was causing an error. I can't think of any case where it would get us in trouble 🤷